packages/config/README.md

```diff
@@ -149,7 +149,8 @@ console.log(configs); // all found configs from all dirs
     "levels": [
         {
             "path": "path/to/level",
-            "scheme": "nested"
+            "scheme": "nested",
+            "naming": "react"
```

Comment:
can I defined `scheme` and `naming` on the top near the `root`-prop? Simple example "I want to define react-naming to all my levels"

Reply:
You promised to write them on the top [here](https://github.com/bem/bem-sdk/issues/277#issuecomment-368453430) ))

Reply:
oh, i misunderstood your idea.
let's doublecheck it again.

for following file structure:
```
parent-dir/
    .bemrc # root: true here
    project-cwd/
        .bemrc
```
and
```js
// parent-dir/.bemrc
{
    "root": true,
    "levels": [
        {
            "path": "path/to/level",
            "scheme": "nested",
            "naming": "react"
        }
    ]
}

// parent-dir/project-cwd/.bemrc
{
    "levels": [
        {
            "path": "path/to/level",
            "scheme": "nested"
        }
    ]
}
```
run
```js
require('@bem/sdk.config')().levelSync('path/to/level'); // { scheme: 'nested', naming: 'react' }
```

isn't it exactly what you need?

Reply:
Mmm. levelSync takes `path` not `layer`. My bad, sorry

Reply:
Will you add `naming/scheme` props on the top level of config like:

```js
{
  "root": true,
  "naming": "react",
  "levels": ...
}
```

Reply:
It works as you wish right now (any option from root level will be available in `levelSync()` output). The reason I don't want it in example in readme is that each level can actually have its own naming scheme.

Reply:
I understand but specification `naming` on top is very common. So I propose to add it in example